src/main/java/net/wasdev/wlp/common/plugins/util/DevUtil.java

```diff
@@ -680,6 +681,10 @@ public void watchFiles(File buildFile, File outputDirectory, File testOutputDire
             debug("Watching build file directory: " + buildFile.getParentFile().toPath());
 
             while (true) {
+                if (serverThread.getState().equals(Thread.State.TERMINATED)){
+                    System.exit(0);
```

Comment:
Is there a way to terminate more gracefully? Perhaps with a `PluginScenarioException`?